src/sourmash/commands.py

```diff
@@ -797,7 +797,8 @@ def gather(args):
             if prefetch_csvout_fp:
                 prefetch_csvout_fp.flush()
 
-        notify(f"Found {len(save_prefetch)} signatures via prefetch; now doing gather.")
+        display_bp = format_bp(args.threshold_bp)
+        notify(f"Prefetch found {len(save_prefetch)} signatures with overlap >= {display_bp}.")
```

Comment:
should we notify anything about https://github.com/sourmash-bio/sourmash/issues/2318?

Reply:
I think linking this PR into that issue is enough? :) yeah that requires a whole different set of foo!